facility-all-programs.ipynb

```diff
@@ -327,7 +475,7 @@
     "        print( \"Data from RCRA_VIOLATIONS stored in rcra_viol_data.\")\n",
     "        have_rcra_viol = True\n",
     "    except pd.errors.EmptyDataError:\n",
-    "        print( \"No data for this facility in RCRA_VIOLATIONS.\")\n",
+    "        print( \"This is a RCRA-regulated facility, but there's no data for it in RCRA_VIOLATIONS.\")\n",
```

Comment:
instead of "there's no data for it in RCRA_VIOLATIONS" would it be accurate to say "there's no publicly available data about its RCRA violation history"? Would be clearer

Reply:
I agree it would be clearer, but if I put my nitpicky hat on, there may actually be publicly available data about its violation history _somewhere_. There's no data for it in this spreadsheet seems most conservative? But I can reword to make it clear we're referring to this ECHO spreadsheet/table.